IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcTranslationOperation/DocEntity.xml

```diff
@@ -1,9 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcTranslationOperation" UniqueId="946a945b-8a4a-4e2c-9840-b56032727b5f" BaseDefinition="IfcCoordinateOperation" EntityFlags="32">
 	<Attributes>
-		<DocAttribute Name="Eastings" UniqueId="f2534a2f-e119-4a52-b1cf-a55c170eea45" DefinedType="IfcLengthMeasure" />
-		<DocAttribute Name="Northings" UniqueId="366b1fda-0ecf-4891-abb5-212d322b50fb" DefinedType="IfcLengthMeasure" />
-		<DocAttribute Name="OrthgonalHeight" UniqueId="caad8793-db34-4377-b751-b1591d8f3fe3" DefinedType="IfcLengthMeasure" AttributeFlags="1" />
+		<DocAttribute Name="FirstCoordinate" UniqueId="f2534a2f-e119-4a52-b1cf-a55c170eea45" DefinedType="IfcMeasureValue">
+			<Documentation>The first coordinate of translation. Can be a length measure in case of map coordinates and angle in case of geographic reference systems.</Documentation>
```

Comment:
```suggestion
			<Documentation>The first coordinate of translation. Can be a length measure in case of map coordinates and solid angle measure in case of geographic coordinates.</Documentation>
```

Reply:
Maybe I reference the exact defined types?